lib/web_console/middleware.rb

```diff
@@ -19,12 +14,15 @@ class Middleware
 
     UNACCEPTABLE_REQUEST_MESSAGE = "A supported version is expected in the Accept header."
 
+    cattr_accessor :mount_point
+    @@mount_point = '/__web_console'
```

Comment:
Maybe this can be a small task for the future. I think we can convert all those default settings to use `self.` instead of `@@` so we don't rely on internal implementation knowledge.